### DISCUSSION TITLE
Minimal Deno Runtime

### DISCUSSION THREAD

**Discussion by yerlaser** [denoland/deno#11475] [Ideas]
Deno includes many tools that are only needed during development and not by users of scripts, e.g. fmt, lint, compile, bundle etc.
So, I suggest to publish an slimmed down Deno runtime for purely running scripts.
This can be useful if a large company with strict security requirements wants to distribute such a runtime to all server and desktops.
The slimmer runtime will have less bugs in it and thus more secure.

**Comment by elycheikhsmail** [comment]
I think this is a good approch, the only problem with it that will broken all existing code. I think we will create repo deno-tools for deno tools test fm ....and deno-min cli for deno minimal cli.
deno dev focus on deno-min and use denotools for getting actual deno.
